andela/technites-bn-backend
# #168748854 Errors should be handled well and logged in a log file

#### What does this PR do?
Add an error handler to ensure they are handled well and logged in a log file
#### Description of Task to be completed?
- Add error handler for the development environment and log them in a file
- Add error handler for the production environment giving their statistics of frequency and time of occurrence
#### How should this be manually tested?
- git clone this repo and checkout to `ft-add-error-logger-168748854` branch
- run the application with an error and check the log file in the log folder for the specific error
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168748854](https://www.pivotaltracker.com/story/show/168748854)
#### What are the relevant Github Issues?
N/A
#### Screenshots (if appropriate)
###### sample local log
![image](https://user-images.githubusercontent.com/45405036/65873470-7a161800-e383-11e9-9de1-c1838be32e4d.png)

###### Sentenary log
![image](https://user-images.githubusercontent.com/45405036/65873569-b2b5f180-e383-11e9-9436-eb4eb85fb1ee.png)

#### Questions:
N/A
